pkg/minikube/cruntime/docker.go

```diff
@@ -137,6 +139,7 @@ func (r *Docker) Disable() error {
 func (r *Docker) ImageExists(name string, sha string) bool {
 	// expected output looks like [SHA_ALGO:SHA]
 	c := exec.Command("docker", "image", "inspect", "--format", "{{.Id}}", name)
+	c.Env = os.Environ()
```

Comment:
I don't think this is necessary -- os.Environ() is always passed into exec.

Reply:
https://golang.org/pkg/os/exec/#Cmd
Looks like you're right, I overlooked this bit in the docs.